packages/app/components/Queue/Student/StudentQueueCard.tsx

```diff
@@ -40,9 +42,59 @@ export default function StudentQueueCard({
   return (
     <HorizontalStudentCard>
       <CenterRow>
+        {/* Temporary removal of hotness bar
+        <Col flex="0 1 auto" style={{ margin: '0 6px 0 0' }}>
+          <HotnessBar hotness={Math.floor(Math.random() * 101)} />
+        </Col> */}
+        {isStaff && ( // only show avatar if staff for now. TODO: fix endpoint to allow queues to access student avatars and names if prof enabled it
+          <Col flex="0 1 auto" style={{ margin: '0 12px 0 0' }}>
+            <KOHAvatar
+              size={46}
+              name={question.creator.name}
+              photoURL={question.creator.photoURL}
+            />
+          </Col>
+        )}
         <Col flex="1 1">
-          <Text>{truncate(question.text, 150)}</Text>
-          <div
+          <Tooltip // only show tooltip if text is too long
```

Comment:
We will probably want to change this to a "Click to expand" feature later on.

Reply:
yep this is a good idea. Plus if there's like 4 people who joined a question or something, we can show it in the expanded view or any other details we want to show about a question